doc/syntax.rst

```diff
@@ -0,0 +1,38 @@
+=====================
+Sphinx-Gallery syntax
+=====================
+
+For your examples to work you have to make them valid python code.
+
+
+Simple examples
+===============
+
+This are the base of a python script of module. The structure follows
+a leading docstring that must reStructuredText syntax and a header.
```

Comment:
missing verb between must and reStructuredText. Maybe "must follow reStructuredText".